api/async/stargazers_test.ts

```diff
@@ -0,0 +1,52 @@
+import { assertEquals } from "../../test_deps.ts";
+import {
+  createContext,
+  createScheduledEvent,
+} from "../../utils/test_utils.ts";
+import { handler } from "./stargazers.ts";
+import { Database, Module } from "../../utils/database.ts";
+import { GitHub } from "../../utils/github.ts";
+
+const database = new Database(Deno.env.get("MONGO_URI")!);
+const gh = new GitHub();
+
+const ltest: Module = {
+  name: "ltest",
+  description: "Testing all the things!",
+  type: "github",
+  owner: "luca-rand",
+  repo: "testing",
+  star_count: 0, // real number is 2 atm
+  is_unlisted: false,
+  created_at: new Date(2020, 1, 1),
+};
+
+const utest: Module = {
+  name: "unlisted_module",
+  description: "Testing all the things! -- unlisted",
+  type: "github",
+  owner: "wperron",
+  repo: "testing",
+  star_count: 0,
+  is_unlisted: true,
+  created_at: new Date(2020, 1, 1),
+};
+
+Deno.test({
+  name: "crawl stargazers",
+  async fn() {
+    await database.saveModule(ltest);
+    await database.saveModule(utest);
+
+    await handler(
+      createScheduledEvent(),
+      createContext(),
+    );
+
+    const updated = await database.getModule(ltest.name);
+    assertEquals(updated?.star_count, 2);
```

Comment:
It is too easy to break this test. Maybe `assert(updated?.star_count >= 2)`. That should hold true as long as me and you have starred it :-D

Reply:
yes I know 🤣 do you know of way to lock the number of starts for a repo? I guess worst case scenario we can simply check that the number of stars is greater than 0

Reply:
I would just change it to `assert(updated?.star_count >= 2)` or `assert(updated?.star_count >= 1)`